content/content/teaching_aids/course_management_systems.md

```diff
@@ -0,0 +1,68 @@
+# Samskrit Bharti Courses Management System
```

Comment:
Please change heading to `##`. You may need to adjust subsection levels accordingly. Also, please ensure that there are no spaces before #